### DISCUSSION TITLE
Remote Desktop RDP with a different port

### DISCUSSION THREAD

**Discussion by mindracer13** [simeononsecurity/Windows-Optimize-Harden-Debloat#95] [General]
I run RDP on a different port, after I applied the script I confirmed RDP was listening on the different port, but the firewall rule was no longer working.

I even removed and re-added the firewall rule and still wouldn't accept any incoming connections.

Then I had to disable some firewall GPOs set by the script I imagine, and then I had to add the rule via Computer Policy > Windows Settings > Security Settings > Windows Defender Firewall with Advanced Security > > Inbound Rules.

Once i added in the rule in there, then it started accepting incoming connections.

I'm a little confused, using this script, does the normal Windows Defender Advanced Firewall configuration no longer work?

**Comment by simeononsecurity** [comment]
> I run RDP on a different port, after I applied the script I confirmed RDP was listening on the different port, but the firewall rule was no longer working.
> 
> I even removed and re-added the firewall rule and still wouldn't accept any incoming connections.
> 
> Then I had to disable some firewall GPOs set by the script I imagine, and then I had to add the rule via Computer Policy > Windows Settings > Security Settings > Windows Defender Firewall with Advanced Security > > Inbound Rules.
> 
> Once i added in the rule in there, then it started accepting incoming connections.
> 
> I'm a little confused, using this script, does the normal Windows Defender Advanced Firewall configuration no longer work?

From our testing it works fine. This is one of our more heavily tested configurations, which should primarily just strengthen the firewall settings. If you can provide more details we'd be glad to look into it further. We're pretty certain things are working as intended there however.
